spring-petclinic-microservices/spring-petclinic-vets-service/pom.xml

```diff
@@ -7,6 +7,7 @@
 	<artifactId>spring-petclinic-vets-service</artifactId>
 	<packaging>jar</packaging>
 	<description>Spring PetClinic Vets Service</description>
+    <name>Spring Petclinic Vets Service</name>
```

Comment:
For some historical reasons, 
for petclinc samples,
I suggest we focus the names here,
use other issue to do this, since the petclinic wii refactor soon.